src/components/scss/_navigation.scss

```diff
@@ -11,4 +11,120 @@
   &:not(:last-of-type) {
     margin-right: 2rem;
   }
-}
\ No newline at end of file
+}
+
+.navbar-burger {
+  display: block;
+  
+  @include media(md){
+    display: none;
+  }
+}
+
+.burger{
+    width: 50px;
+    height: 50px;
+    display: flex;
+    cursor: pointer;
+    justify-content: center;
+    #bar1,#bar2,#bar3{
+      position: absolute;
+      width: 35px;
+      height: 4px;
+      background-color: #150e60;
+      margin: 6px 0;
+      transition: 0.4s;
+      border-radius: 4px;
+    }
+    #bar2{
+      top: 31px;
+    }
+    #bar3{
+      top: 46px;
+    }
+  @include media(md){
+    display: none;
+  }
+    
+}
+
+.is-active {
+   #bar1{
+    -webkit-transform: rotate(-45deg) translate(-5px, 13px);
+    transform: rotate(-45deg) translate(-5px, 13px);
+   }
+   #bar2{
+     opacity: 0;
+     -webkit-transform: translate(20px, 0px);
+      transform: translate(20px, 0px);
+   }
+   #bar3{
+    -webkit-transform: rotate(45deg) translate(-8px, -16px);
+    transform: rotate(45deg) translate(-8px, -16px);
+   }
+}
+
+.dropDownMenu{
+   width: 98vw;
+   transform: translateX(101%);
+   display: flex;
+   flex-direction: column;
+   justify-content: space-between;
+   align-items: center;
+   height: 90vh;
+   position: absolute;
+   top: 80px;
+   right: 0;
+   background: white;
+   text-align: center;
+   padding: 20px;
+   transition: .5s ease-in-out;
+a{
+    padding: 20px;
+    font-weight: 700;
+ }
+.button{
+    width: 50%;
+    margin-bottom: 20px;
+}
+}
+  .is-active{
+   
+    .dropDownMenu{
+      @include media(sm){
+        width: 50vw;
+      }
+       width: 100vw;
+       transform: translateX(0%);
+       display: flex;
+       flex-direction: column;
+       justify-content: space-between;
+       align-items: center;
+       height: 90vh;
+       box-shadow: -50vw 0vw 0vw 0 #000000a1;
+       position: fixed;
+       top: 80px;
+       right: 0;
+       background: white;
+       text-align: center;
+       padding: 20px;
+       transition: .5s ease-in-out;
+      a{
+        padding: 20px;
+        
+        font-weight: 700;
+      }
+      .button{
+        width: 50%;
+        margin-bottom: 20px;
+       
+      }
+    }
+  }
+
+ 
+
+
+
+  
+
```

Comment:
Be sure to install your own local formatters and linters. We have our own set of configs for prettier and eslint that should be followed so we're not introducing simple formatting issues.